src/test/java/com/notably/logic/suggestion/block/BlockTreeItemStub.java

```diff
@@ -0,0 +1,97 @@
+package com.notably.logic.suggestion.block;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.notably.model.block.Block;
+import com.notably.model.block.BlockImpl;
+import com.notably.model.block.BlockTreeItem;
+import com.notably.model.block.Body;
+import com.notably.model.block.Title;
+
+import javafx.scene.control.TreeItem;
+
+/**
+ * A Stub class for BlockTreeItem.
+ */
+public class BlockTreeItemStub implements BlockTreeItem {
```

Comment:
I don't think you need this stub! Firzan has already implemented `BlockTreeItemImpl`, so you can just create those